src/components/assets/modals/ModalXcmBridge.vue

```diff
@@ -172,8 +183,24 @@ export default defineComponent({
   },
   setup(props) {
     const isClosingModal = ref<boolean>(false);
+    const isMobileDisplayTooltip = ref<boolean>(false);
     const token = computed(() => props.token);
 
+    const isDisplayTooltip = computed<boolean | null>(() => {
```

Comment:
I noticed this code block is repeating 3 times in different files. Can you refactor so the logic is defined only once?

Reply:
Sure, I will create a custom hook then.

Reply:
@bobo-k2 
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/451/commits/29d0b5b531e241f02561932129c2a799ade187d2) commit.

Reply:
@bobo-k2 Please help take a look my comments.